crates/goose-mcp/src/computercontroller/xlsx_tool.rs

```diff
@@ -322,10 +322,21 @@ mod tests {
         assert_eq!(data_cell.value, "Canada");
         assert!(data_cell.formula.is_none());
 
+        // Test B1 cell (known value from FinancialSample.xlsx)
+        let b1_cell = xlsx.get_cell_value(worksheet, 1, 2)?;
+        assert_eq!(b1_cell.value, "Country");
+        assert!(b1_cell.formula.is_none());
+
+        // Test A2 cell (known value from FinancialSample.xlsx)
+        let a2_cell = xlsx.get_cell_value(worksheet, 2, 1)?;
```

Comment:
remove the comments about known value. if you want to say something about this, change get_cell_value to take a string and do the row col parsing there, which would make the test easier to read